api/swagger.yaml

```
@@ -8331,6 +8331,15 @@ paths:
           description: "BuildKit output configuration"
           type: "string"
           default: ""
+        - name: "version"
+          in: "query"
+          type: "string"
+          default: "1"
+          description: |
+            Version of the underlying builder to use.
+
+            "1" is the first generation classic builder in docker daemon (default).
+            "2" is a moby/buildkit based builder.
```

just some nit suggestions

```suggestion
            Version of the builder backend to use.

            - `1` is the first generation classic builder in the Docker daemon (default)
            - `2` is [BuildKit](https://github.com/moby/buildkit)
```

---

Thanks, changed!